backend/src/main/java/org/latexscribe/LatexScribe/webcontroller/TagController.java

```diff
@@ -0,0 +1,34 @@
+package org.latexscribe.LatexScribe.webcontroller;
+
+import lombok.RequiredArgsConstructor;
+import org.latexscribe.LatexScribe.domain.model.DocumentTag;
+import org.latexscribe.LatexScribe.service.ITagService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/tags")
+public class TagController {
+    private final ITagService tagService;
+
+    @GetMapping
```

Comment:
```suggestion
    @GetMapping("/")
```

I know `GetMapping` without arguments uses the root, but I would still prefer to have an explicit definition.